pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -31,7 +31,7 @@ import (
 
 var negPriority, lowPriority, midPriority, highPriority, veryHighPriority = int32(-100), int32(0), int32(100), int32(1000), int32(10000)
 var mediumPriority = (lowPriority + highPriority) / 2
-var highPriorityPod, highPriNominatedPod, medPriorityPod, unschedulablePod = v1.Pod{
+var highPriorityPod, highPriNominatedPod, medPriorityPod, unschedulableLowPriorityPod = v1.Pod{
```

Comment:
Could you please revert the name of the pod back? It has created a large number of diff lines unrelated to this PR.